<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html
	PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
 <head>
  <title>Cell Circuit Search : Haugen et al. (2004)</title>
  <link rel="stylesheet" type="text/css" href="/master.css" />
 </head>
<body>



<h2>FAQ: Haugen et al. (2004)</h2> 

"Integrating phenotypic and expression profiles to map arsenic-response networks"
<a class="white-bg-link" href="http://genomebiology.com/2004/5/12/R95">[fulltext]</a>
   
<h2>Original figures from this paper</h2><a href="./fig1_lrg.jpg"><img src="./fig1_small.jpg" align="left" hspace="10">Figure 1.
Arsenic-induced signaling and regulatory mechanisms involve transcriptional activators and the proteasome. (a-d) Significant network neighborhoods (p < 0.005) uncovered by the ActiveModules algorithm, with the search performed at depth 1 (all nodes in the network are the nearest neighbors of one central node): (a) FHL1 center; (b) PRE1 center and proteasome complex; (c) YAP1 and CAD1 centers; (d) HSF1 center. (e) An additional network centered on MET31 with functional relevance to the arsenic response, which, however, did not reach significance in this analysis, p < 0.11. (f) An overview of the network relationships between major arsenic-responsive transcription factors. Shades of red, induced; shades of green, repressed; blue boxed outline, significant expression; orange arrows, protein-DNA interaction; blue dashed lines, protein-protein interactions. The 2 h, 100 micro Molar AsIII condition was used for the visual mappings. Many of the genes mapped to the network neighborhoods and displayed in this figure are boxed for the sake of clarity and space, but are mostly significantly differentially expressed.<br clear=left><br><a href="./fig5_lrg.jpg"><img src="./fig5_small.jpg" align="left" hspace="10">Figure 5.
LinearActivePaths analysis finds that virtually all genes in active metabolic networks confer sensitivity to arsenic when deleted. (a) Serine, threonine, glutamate amino-acid synthetic pathways; (b) the shikimate pathway. The paths that compose these networks all have individual p-values of < 0.05. The coloration for these figures is based on red for any gene ranked in the top 50 significant genes, yellow for 51-100, and green for >101.
<br clear=left><br>

<h2>Q&A</h2>
<ol>
<li><b>What is this paper about?</b>
<p>
<table width="80%">
<tr><td>
<b>Abstract:</b> BACKGROUND: Arsenic is a nonmutagenic carcinogen affecting millions of people. The cellular impact of this metalloid in Saccharomyces cerevisiae was determined by profiling global gene expression and sensitivity phenotypes. These data were then mapped to a metabolic network composed of all known biochemical reactions in yeast, as well as the yeast network of 20,985 protein-protein/protein-DNA interactions. RESULTS: While the expression data unveiled no significant nodes in the metabolic network, the regulatory network revealed several important nodes as centers of arsenic-induced activity. The highest-scoring proteins included Fhl1, Msn2, Msn4, Yap1, Cad1 (Yap2), Pre1, Hsf1 and Met31. Contrary to the gene-expression analyses, the phenotypic-profiling data mapped to the metabolic network. The two significant metabolic networks unveiled were shikimate, and serine, threonine and glutamate biosynthesis. We also carried out transcriptional profiling of specific deletion strains, confirming that the transcription factors Yap1, Arr1 (Yap8), and Rpn4 strongly mediate the cell's adaptation to arsenic-induced stress but that Cad1 has negligible impact. CONCLUSIONS: By integrating phenotypic and transcriptional profiling and mapping the data onto the metabolic and regulatory networks, we have shown that arsenic is likely to channel sulfur into glutathione for detoxification, leads to indirect oxidative stress by depleting glutathione pools, and alters protein turnover via arsenation of sulfhydryl groups on proteins. Furthermore, we show that phenotypically sensitive pathways are upstream of differentially expressed ones, indicating that transcriptional and phenotypic profiling implicate distinct, but related, pathways.
</td>
</tr>
</table>
<br>
<li><a name="organisms"><b>What organism is being modeled?</b></a>
<p>
Yeast <i>(Saccharomyces cerevisiae)</i>
<p>
</ol>
</body>
</html>
  
