<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html
	PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
 <head>
  <title>Cell Circuit Search : Ideker et al. (2002)</title>
  <link rel="stylesheet" type="text/css" href="/master.css" />
 </head>
<body>



<h2>FAQ: Ideker et al. (2002)</h2> 

"Discovering regulatory and signalling circuits in molecular interaction networks"
<a class="white-bg-link" href="http://bioinformatics.oxfordjournals.org/cgi/reprint/18/suppl_1/S233">[fulltext]</a>
   
<h2>Original figures from this paper</h2><a href="./fig5_lrg.jpg"><img src="./fig5_small.jpg" align="left" hspace="10">Figure 5: Performance on a large interaction network using multiple conditions. The seven highest scoring subnetworks screened from the large network are shown in decreasing order of score. Subnet 1, containing 340 nodes, was subjected to a second application of simulated annealing to produce five smaller subnets 1a-1e. Subnetworks are represented as in Fig. 1, but with protein->DNA interactions denoted by yellow edges and protein-protein interactions denoted by blue edges.
<br clear=left><br>

<h2>Q&A</h2>
<ol>
<li><b>What is this paper about?</b>
<p>
<table width="80%">
<tr><td>
<b>Abstract:</b> MOTIVATION: In model organisms such as yeast, large databases of protein-protein and protein-DNA interactions have become an extremely important resource for the study of protein function, evolution, and gene regulatory dynamics. In this paper we demonstrate that by integrating these interactions with widely-available mRNA expression data, it is possible to generate concrete hypotheses for the underlying mechanisms governing the observed changes in gene expression. To perform this integration systematically and at large scale, we introduce an approach for screening a molecular interaction network to identify active subnetworks, i.e., connected regions of the network that show significant changes in expression over particular subsets of conditions. The method we present here combines a rigorous statistical measure for scoring subnetworks with a search algorithm for identifying subnetworks with high score. RESULTS: We evaluated our procedure on a small network of 332 genes and 362 interactions and a large network of 4160 genes containing all 7462 protein-protein and protein-DNA interactions in the yeast public databases. In the case of the small network, we identified five significant subnetworks that covered 41 out of 77 (53%) of all significant changes in expression. Both network analyses returned several top-scoring subnetworks with good correspondence to known regulatory mechanisms in the literature. These results demonstrate how large-scale genomic approaches may be used to uncover signalling and regulatory pathways in a systematic, integrative fashion.
</td>
</tr>
</table>
<br>
<li><a name="organisms"><b>What organism is being modeled?</b></a>
<p>
Yeast <i>(Saccharomyces cerevisiae)</i>
<p>
</ol>
</body>
</html>
  
